lib/Doctrine/DBAL/Driver/AbstractOracleDriver.php

```diff
@@ -136,7 +136,7 @@ protected function getEasyConnectString(array $params)
                 $service = 'SERVICE_NAME=' . $serviceName;
             }
 
-            if (isset($params['instancename']) && ! empty($params['instancename'])) {
```

Comment:
You can remove this isset entirely.

Reply:
I did some tests and this `empty`s are valid. Gonna close it :smile:

Reply:
Yes, but still, these issets are redundant. :)

Reply:
Gonna study more and PR this separately :man_technologist: